### PR TITLE
Add F-Secure Anti-Virus v16.4

### PR DESCRIPTION
**Pull request by dcondrey**
Note that the install is very slow.  It takes several minutes to complete but it does complete successfully.  I do not know why it takes so long.

*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/f-secure-anti-virus.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask 'f-secure-anti-virus' do
+  version ':latest'
```
Comment (vitorgalvao) [inline Casks/f-secure-anti-virus.rb]: `':latest'` is invalid. You mean `:latest` (no quotes).

File: Casks/f-secure-anti-virus.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  sha256 'bd22b8841b8ebbc178936afbf9a6c6668750e1cfc313500419fe4d7b5960542c'
```
Comment (vitorgalvao) [inline Casks/f-secure-anti-virus.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

File: Casks/f-secure-anti-virus.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+  url 'https://download.sp.f-secure.com/SE/Retail/installer/F-Secure-Anti-Virus-for-Mac.mpkg'
+  name 'F-Secure Anti-Virus'
+  homepage 'https://www.f-secure.com/en/web/home_global/anti-virus'
+
+  pkg 'F-Secure-Anti-Virus-for-Mac.mpkg'
+
+  uninstall script: '/usr/local/f-secure/bin/uninstall_MacProtection'
```
Comment (vitorgalvao) [inline Casks/f-secure-anti-virus.rb]: Add a `pkgutil` in addition to the script.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/f-secure-anti-virus.rb (hunk 3 above)
Comment (miccal) [inline Casks/f-secure-anti-virus.rb]: Using the commands listed [here](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md#working-with-a-pkg-file-manually), you need to add the following to the `pkgutil:`:
```
com.f-secure.agents-and-daemons
com.f-secure.aua
com.f-secure.fsmac
com.f-secure.fsmac.bin
com.f-secure.fsmac.customization
com.f-secure.fsmac.fsauth
com.f-secure.fsmac.gui
com.f-secure.fsmac.gui.customization
com.f-secure.fsmac.lib
com.f-secure.fsmac.tools
com.f-secure.fssp
com.f-secure.orspclient
```

Comment (miccal) [inline Casks/f-secure-anti-virus.rb]: Investigating the `.mpkg`, I think the `uninstall script:` also requires `sudo` - see an example [here](https://github.com/caskroom/homebrew-cask/blob/master/Casks/drobo-dashboard.rb).

@dcondrey please confirm that `brew cask uninstall f-secure-anti-virus` does indeed work.

**Comment by dcondrey** [comment]
Updated based on feedback and uninstall does complete successfully.

**Review by miccal — APPROVED** [review]

**Comment by vitorgalvao** [comment]
Thank you.

**Comment by dcondrey** [comment]
Thank you.

**Pull request merged**